Dockerfile

```diff
@@ -15,11 +15,14 @@ FROM $BASE_IMAGE AS ironic-builder
 # ``Cannot initialize '::'``
 # This is due to the conversion table missing codepage 850, included in glibc-gconv-extra
 RUN dnf install -y gcc git make xz-devel glibc-gconv-extra
+
 WORKDIR /tmp
+
 RUN git clone --depth 1 --branch v1.21.1 https://github.com/ipxe/ipxe.git && \
       cd ipxe && \
       cd src && \
-      make bin/undionly.kpxe bin-x86_64-efi/ipxe.efi bin-x86_64-efi/snponly.efi
+      # NOTE(elfosardo): warning should not be treated as errors by default
```

Comment:
I wonder if there is a newer version without warnings. Not in this PR though.

Reply:
I did a test today and unfortunately this behavior has not changed :/